aptos-move/aptos-vm/src/block_executor/storage_wrapper.rs

```diff
@@ -56,6 +56,12 @@ impl<'a, S: StateView> StateView for VersionedView<'a, S> {
                     .map_err(|pe| pe.finish(Location::Undefined).into_vm_status())?;
                 Ok(Some(serialize(&result)))
             }
+            // TODO: generate parallel execution error for StatusCode.
```

Comment:
Actually, what we need in this case is to tell the VM to halt immediately (can even avoid doing Epilogue, etc), and to not log anything (on the VM side). We can then independently decide on our end whether we would like to log anything or not.

Storage error might be a blocker to landing this as is, because it will log unrelated error messages and confuse everybody. So either we are blocked on logging refactoring (to only actually log the last execution logs), or getting the above feature from the VM which seems independently useful anyway. @runtian-zhou : can you advice how hard would it be to support that kind of behavior? 
(new error that indicates parallel execution is asking to immediately halt with minimal overhead and return with any status from the transaction execution / session, as parallel execution knows this execution is gonna get ignored).

Reply:
This is tricky because it will be converted into a storage error from Move side. Basically MoveVM will convert this error into a storage error and return things back...Maybe a better thing to do here is to just return a `None`?